### DISCUSSION TITLE
No "plover_plugins" script on Raspberry Pi with 4.0.0.dev12  (installed via Python wheel)

### DISCUSSION THREAD

**Discussion by tkyler** [opensteno/plover#1615] [General]
Installed 4.0.0.dev12 on Raspberry Pi using the Python wheel and pip3...installed and runs great, but no plugins manager.  Tried to install scripts  manually via the command line, but error message,  "no such script:  plover_plugins" results.  Possible to manually install the plugins manager?  I'm not terribly smart on Python deployment structures.

**Comment by user202729** [comment]
`plover_plugins` is provided by the plugin manager.

if you have access to `pip` then you can simply write `pip3 install <package-name>`

TODO update the page https://github.com/openstenoproject/plover/wiki/Manually-Installing-Plugins
